js/api/types/brevTypes.ts

```diff
@@ -0,0 +1,19 @@
+export interface Brev {
+  uuid: string;
+  deltakerUuid: string;
+  createdAt: string;
+  brevType: string;
+  digitalt: boolean;
+  lestDato?: string;
+  fritekst: string;
+  sted: string;
+  tid: string;
+  videoLink?: string;
+  document: BrevDocument[];
+  virksomhetsnummer: string;
+}
+
+type BrevDocument = {
+  type: string;
```

Comment:
Type er en av disse; HEADER, PARAGRAPH, LINK

Foreslår å lage en typescript type for disse.

Reply:
Her er også header, som vi må ha med. Ellers blir det nok feil i vår visning:

https://github.com/navikt/isdialogmote/blob/master/src/main/kotlin/no/nav/syfo/dialogmote/domain/DocumentComponentDTO.kt

Reply:
Ellers synes jeg kanskje DocumentComponent (som den heter i backend) er mer beskrivende enn BrevDocument. 

Dette er i praksis bestanddeler av et brev/dokument

Reply:
Dette er kopiert fra dialogmotearbeidsgiver, men jeg er enig i det du påpeker. Jeg foreslår at jeg fikser det nå i denne appen og så tar jeg jobben med å fikse det i dialogmotearbeidsgiver samtidig som jeg fikser knapper for respons der.

Reply:
Ulempen med to apper, ja :/

Reply:
Jeg kan fikse det for `dialogmotearbeidsgiver` siden jeg herjer i den appen nå, det var opprinnelig `BrevDocumentComponent` :P

Reply:
> Her er også header, som vi må ha med. Ellers blir det nok feil i vår visning:
> 
> https://github.com/navikt/isdialogmote/blob/master/src/main/kotlin/no/nav/syfo/dialogmote/domain/DocumentComponentDTO.kt

Hva mener du med dette, @AudunSorheim? At vi må ha med header som en av DocumentComponentType? Eller noe annet?

Reply:
beklager, det var "title" jeg mente, ikke header. Title blir brukt i isyfo til å vise en tittel for den gitte bestanddelen av dokumentet. Ser at vi mangler den i vår typedefinisjon, ref lenke til isdialogmote. Vet ikke om vi bruker title i vår visning, men om ikke har vi nok en bug.

Reply:
Tror jeg har fikset DocumentComponent og title nå :)